File: pgm/ABC_Visual_Basic/ABC_136_B.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N As Integer = Console.ReadLine()
        Dim ans As Integer = 0
        For i As Integer = 1 To N
            If (Math.Floor(Math.Log10(i)) + 1) Mod 2 = 1
                ans += 1
            End If
        Next
        Console.WriteLine(ans)
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_136_B.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions

class Program
{
    static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int ans = 0;
        for (int i = 1; i < N + 1; i++)
            if ((Math.Floor(Math.Log10(i)) + 1) % 2 == 1) ans++;
        Console.WriteLine(ans);
        return;
    }
}